notebooks/README.md

```diff
@@ -0,0 +1,131 @@
+# "notebooks" directory
+This directory stores all Jupyter Notebook files that have been used as the first step of creating this project. It is 
+divided into two subdirectories: `data_analysis` and `models`.
+
+## Table of contents
+* `data_analysis` - contains files used to analyse data about monsters from Pathfinder and roughly estimate which parts 
+of it will not be used in finished project. 
+  * [abomination_vaults_bestiary_analysis.ipynb](#data_analysisabomination_vaults_bestiary_analysisipynb)
+  * [pathfinder_analysis_of_other_columns.ipynb](#data_analysispathfinder_analysis_of_other_columnsipynb)
+  * [pathfinder_system_abilities_attributes.ipynb](#data_analysispathfinder_system_abilities_attributesipynb)
+  * [pathfinder_system_other_columns.ipynb](#data_analysispathfinder_system_other_columnsipynb)
+* `models` - contains machine learning models.
+  * [system_linear_regression.ipynb](#modelssystem_linear_regressionipynb)
+
+
+## data_analysis/abomination_vaults_bestiary_analysis.ipynb
+Analysis of `pathfinder_2e_data/abomination-vaults-bestiary.db`.
```

Comment:
Ten notebook(to jakiś mój stary) chyba pasuje usunąć i w przyszłości zrobić analizę potworów spoza bestiariuszy, ale to już po zrobieniu tych zadań datasetowych

Reply:
Chyba można zostawić, zwłaszcza jak sama piszesz, że w przyszłości by to trzeba było zrobić od nowa